package.json

```diff
@@ -1,36 +1,28 @@
 {
   "name": "frontend",
   "version": "0.1.0",
+  "proxy": "http://localhost:8000",
   "private": true,
   "homepage": ".",
-  "proxy": "http://localhost:8000",
   "dependencies": {
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.2.5",
     "@mui/material": "^5.2.6",
+    "@react-google-maps/api": "^2.7.0",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "http-proxy-middleware": "^2.0.3",
```

Comment:
this is no longer needed. Remove and re-generate package.lock

Reply:
I think this is the last change, I never saw the file that was supposed to be using it make it into a commit.

Reply:
why was proxy removed?

Reply:
It was my error... I removed it but I put it back